packages/helper-react/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "helper-react",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint src --ext ts,tsx --report-unused-disable-directives --max-warnings 0 --resolve-plugins-relative-to ."
```

Comment:
Something I've learned:

I added `--resolve-plugins-relative-to .` to the eslint command because eslint was traveling up to the root of this repository to gather all the eslint configs. By applying this parameter, it forces eslint to only look for the configs and plugins within this folder.

Reply:
Would it be possible to set `"root": true` in the eslint config file? https://eslint.org/docs/latest/use/configure/configuration-files#cascading-and-hierarchy